src/api/index.ts

```diff
@@ -1,21 +1,14 @@
-import { loadToken } from "api/token";
 import { Product } from "../types/inventory";
 
-export const CLIENT_ID = encodeURIComponent(
-  process.env.REACT_APP_CLIENT_ID || ""
-);
-export const CLIENT_SECRET = encodeURIComponent(
-  process.env.REACT_APP_CLIENT_SECRET || ""
-);
+export const LOGIN_URL = (code: string, rfid: string): string =>
+  `https://nibble4-validator.azurewebsites.net/api/login?code=${code}&&rfid=${rfid}`;
```

Comment:
Should make `https://nibble4-validator.azurewebsites.net/api/` into a base variable.